### PR TITLE
EG-638, feedback links and footer translatability

### PR DESCRIPTION
**Pull request by cmrhmrc**

### CODE REVIEW

File: app/views/components/govukFooter.scala.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+ *@
+
+@this()
+
+@(params: Footer = Footer())(implicit messages: Messages)
+
+@import uk.gov.hmrc.govukfrontend.views.Utils._
+@import play.twirl.api.HtmlFormat
+@import params._
+s
```
Comment (woollett-r) [inline app/views/components/govukFooter.scala.html:24]: What is this for?

File: test/base/BaseSpec.scala [hunk 2]
```diff
@@ -19,9 +19,16 @@ package base
...
 import org.scalatest.{MustMatchers, WordSpec}
 import play.api.libs.json._
+import play.twirl.api.Html
+import org.jsoup._
+import org.jsoup.nodes.{Document, Element}
+import scala.collection.JavaConverters._
 
 trait BaseSpec extends WordSpec with MustMatchers with ScalaFutures {
 
+  def asDocument(html: Html): Document = Jsoup.parse(html.toString)
```
Comment (woollett-r) [inline test/base/BaseSpec.scala:29]: I would have put these two methods in ViewSpec as their use is mainly confined to tests of views.

File: app/views/components/govukFooter.scala.html (hunk 1 above)
Comment (cmrhmrc) [reply app/views/components/govukFooter.scala.html]: :) its gone now

File: test/base/BaseSpec.scala (hunk 2 above)
Comment (cmrhmrc) [reply test/base/BaseSpec.scala:29]: will move

File: conf/messages.cy [hunk 3]
```diff
@@ -2,15 +2,39 @@ service.name = Welsh: External guidance
 hello.world = Helo o''r cais view-external-guide-frontend!
 hello.world.currentLanguage = Y gosodiad iaith cyfredol yw "{0}",
 
+service.phaseName = beta
```
Comment (woollett-r) [inline conf/messages.cy:5]: Where possible I'd use the translations from the coronavirus project.

Comment (cmrhmrc) [reply conf/messages.cy:5]: all in

**Pull request merged**